### ISSUE TITLE
ForeignObjectRendering not rendering images and fonts since 1.0.0 rc2

### ISSUE THREAD

**Issue by matteosaporiti** [niklasvh/html2canvas#2031]
Please make sure you are testing with the latest [release of html2canvas](https://github.com/niklasvh/html2canvas/releases). 
Old versions are not supported and issues reported for them will be closed.

# Please follow the general troubleshooting steps first:

- [x] You are using the latest [version](https://github.com/niklasvh/html2canvas/releases)
- [x] You are testing using the non-minified version of html2canvas and checked any potential issues reported in the console

### Bug reports:

Between version 1.0.0 rc1 and 1.0.0 rc2 html2canvas stopped being able to include fonts and images in a saved screenshot with ForeignObjectRendering set as true.
In the network tab I can see the fonts and the images being fetched but they are not included in the final result.
This is the code that I'm using:
```
        html2canvas(document.querySelector("#capture"), {foreignObjectRendering: true, logging: false, proxy: '/proxy'}).then(canvas => {
            download(canvas.toDataURL('image/png'), "screenCapture.png", "image/png"); //download.js used here
        });
```

### Specifications:

 * html2canvas version tested with: 1.0.0 rc1, 1.0.0 rc2, 1.0.0 rc5
 * Browser & version: Chrome 77 and Firefox 69
 * Operating system: Windows 10 build 1903

**Comment by mrcw** [comment]
Yes, if ForeignObjectRendering is configured as true, images are not working... this is a bug.

I've made an online example for this: https://playcode.io/482330?tabs=index.html,preview,console

**Comment by mikr96** [comment]
Hey guys, there is a workaround to this issue, just simply convert the image to base64 format. The package couldn't render images but it can draw. Besides base64, you can also try svg format. Hope this helps! 

![Capture](https://user-images.githubusercontent.com/34701021/98270583-fa1ca180-1fc9-11eb-925d-adff34f7c883.PNG)

**Comment by shidcordero** [comment]
any update on this one? I am also having an issue with this, the font and local image was not rendered

**Comment by meganathan-codenatives** [comment]
@mikr96 Thanks, It's working.........

**Comment by meganathan-codenatives** [comment]
> Hey guys, there is a workaround to this issue, just simply convert the image to base64 format. The package couldn't render images but it can draw. Besides base64, you can also try svg format. Hope this helps!
> 
> ![Capture](https://user-images.githubusercontent.com/34701021/98270583-fa1ca180-1fc9-11eb-925d-adff34f7c883.PNG)

@mikr96 Thanks, It's working.........

**Comment by bastianremo** [comment]
> > Hey guys, there is a workaround to this issue, just simply convert the image to base64 format. The package couldn't render images but it can draw. Besides base64, you can also try svg format. Hope this helps!
> > ![Capture](https://user-images.githubusercontent.com/34701021/98270583-fa1ca180-1fc9-11eb-925d-adff34f7c883.PNG)
> 
> @mikr96 Thanks, It's working.........

how you did it ? I need the solution, I transformed the image to base64 but the page show blank
